docs/crud.md

```diff
@@ -0,0 +1,172 @@
+# General
+
+### Typing
+
+The notion of document type does not exist in couchdb.
+Cozy-stack introduce this notion with a special `_type` field. This type cannot contain "/", it should be unique among all developers, it is recommend to use the Java naming convention with a domain you own.
+All CozyCloud types will be prefixed by io.cozy
+
+------------------------------------------------------------------------------
+
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/io.cozy.events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
+```
+
+#### Response OK
+```http
+HTTP/1.1 200 OK
+Date: Mon, 27 Sept 2016 12:28:53 GMT
+Content-Length: ...
+Content-Type: application/json
+Etag: "3-6494e0ac6494e0ac"
+```
+```json
+{
+    "_id": "io.cozy.events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee",
```

Comment:
Let's make it a good practice: if a developer want to force an id, it should prefix it with the type. But for uuid, no need to keep the _type.